exonum/src/blockchain/transaction.rs

```diff
@@ -147,10 +154,19 @@ pub trait Transaction: Message + ExonumJson + 'static {
 }
 
 /// Result of unsuccessful transaction execution.
+///
+/// An execution error consists
```

Comment:
Is that verbose description of error code + description needed? I guess most of developers are pretty familiar with such a things. The most important information here is that error code affects the state hash and description does not.